package.json

```diff
@@ -22,7 +22,7 @@
     "now-start": "npm run -s stage:venia",
     "prepare": "node ./scripts/update_repo_environment.js && npx lerna bootstrap --hoist",
     "prettier": "prettier --write '@(packages|scripts)/**/*.@(js|css)' '*.js'",
-    "prettier:check": "prettier --list-different '@(packages|scripts)/**/*.@(js|css)' '*.js'",
+    "prettier:check": "prettier-check '@(packages|scripts)/**/*.@(js|css)' '*.js'",
```

Comment:
Maybe I miss something, but we have already added prettier-check package to package.json file, `prettier-check` is an equivalent of `prettier --list-different`.